beamer/agent.py

```diff
@@ -16,6 +17,14 @@
 log = structlog.get_logger(__name__)
 
 
+def _get_contracts_info(config: Config, web3: Web3) -> dict[str, ContractInfo]:
+    chain_id = ChainId(web3.eth.chain_id)
+    info = config.deployment_info.get(chain_id)
+    if info is None:
+        raise RuntimeError(f"Deployment info for chain ID {chain_id} not available")
```

Comment:
can we add the path being used to it? Or at least give a hint where to look at/the problem lies.

Reply:
> can we add the path being used to it? Or at least give a hint where to look at/the problem lies.

Heh, it's not that pretty to add the path in here, I was hoping the "Deployment info" part would be hinting enough
(`--deployment-info`), but generally, it could also be the chain configuration (e.g. wrong RPC). Not sure how to improve this.

Reply:
ok lets go for it now like this